project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.4.9
```

Comment:
sbt 1.5.0 is the latest, but requires play 2.8.8 which was out yesterday.  Otherwise you get a class not found for NSC due to twirl compiler using an internal api of the scala compiler (sbt 1.5.0 uses scala 2.12.13)